app/helpers/vertical_nav_helper.rb

```diff
@@ -0,0 +1,260 @@
+    sections << {id: :subscriptions, title: 'Subscriptions', items: service_subscriptions}       if can?(:manage, :service_plans) && current_account.settings.service_plans_ui_visible?
+    sections << {id: :end_users,     title: 'End-users',     items: service_end_users}           if can?(:manage, :end_users) && current_account.settings.end_user_plans_ui_visible? && !master_on_premises?
+
+    if can? :manage, :plans
+      sections << {id: :ActiveDocs,  title: 'ActiveDocs',  path: admin_service_api_docs_path(@service)}
+      sections << {id: :integration, title: 'Integration', items: service_integration_items}
+    end
+
+    sections
+  end
+
+  def service_analytics
+    items = []
+    items << {id: :usage,              title: 'Usage',              path: admin_service_stats_usage_path(@service)}
+    items << {id: :daily_averages,     title: 'Daily Averages',     path:   admin_service_stats_days_path(@service)}
+    items << {id: :hourly,             title: 'Hourly Averages',    path: admin_service_stats_hours_path(@service)}
+    items << {id: :top_applications,   title: 'Top Applications',   path: admin_service_stats_top_applications_path(@service)}
+    items << {id: :response_codes,     title: 'Response Codes',     path:   admin_service_stats_response_codes_path(@service)}
+    items << {id: :alerts,             title: 'Alerts',             path: admin_service_alerts_path(@service)}
+    # FIXME: this throws an error, verify condition is met and item highlight works
+    items << {id: :integration_errors, title: 'Integration Errors', path: admin_service_errors_path(@service)} if can? :manage, :plans
+    items
+  end
+
+  def service_applications
+    items = []
+    items << {id: :listing,           title: 'Listing',           path: admin_service_applications_path(@service)}
+    items << {id: :application_plans, title: 'Application Plans', path: admin_service_application_plans_path(@service)} if can?(:manage, :plans)
+    items
+  end
+
+  def service_subscriptions
+    items = []
+    items << {id: :subscriptions, title: 'Service Subscriptions', path: admin_buyers_service_contracts_path(search: {service_id: @service.id})}
+    items << {id: :service_plans, title: 'Service Plans',         path: admin_service_service_plans_path(@service)}
+  end
+
+  def service_end_users
+    items = []
+    items << {id: :search,         title: 'Search',         path: admin_service_end_users_path(@service)}
+    items << {id: :end_user_plans, title: 'End-user Plans', path: admin_service_end_user_plans_path(@service)}
+  end
+
+  def service_integration_items
+    items = []
+    items << {id: :configuration,   title: 'Configuration',     path: path_to_service(@service)}
+    items << {id: :methods_metrics, title: 'Methods & Metrics', path: admin_service_metrics_path(@service)}
+    items << {id: :settings,        title: 'Settings',          path: settings_admin_service_path(@service)}
+  end
+
+  # Others
+  def random_sections
```

Comment:
random sections???

Reply:
https://github.com/3scale/porta/pull/859/files/78b5dfbcf4aa746b75b1eb0cd5b8a9e23e221658#diff-f82d17ff82ceac4cebb8068986ecb9e3R11

Reply:
I'd rather call them "miscellaneous_sections" or just "misc_sections" until we find a way to group them better